Packages/com.unity.inputsystem/InputSystem/Editor/InputActionImporterEditor.cs

```diff
@@ -17,6 +17,7 @@ internal class InputActionImporterEditor : ScriptedImporterEditor
     {
         public override void OnInspectorGUI()
         {
+            serializedObject.Update();
```

Comment:
Explain why we need this here. Counterintuitive as ScriptedImporterEditor should be expected to handle this for us.

Add blank line after statement.